core/vm/contracts.go

```diff
@@ -114,6 +115,45 @@ var PrecompiledContractsSHA3FIPS = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{254}): &ecrecoverPublicKey{},
 }
 
+// PrecompiledContractsStaking contains the default set of pre-compiled Ethereum
+// contracts used in the Istanbul release. plus VRF, SHA3FIPS-202 and staking precompiles
+// These are available in the EVM after the StakingPrecompileEpoch
+var PrecompiledContractsStaking = map[common.Address]PrecompiledContract{
+	common.BytesToAddress([]byte{1}): &ecrecover{},
+	common.BytesToAddress([]byte{2}): &sha256hash{},
+	common.BytesToAddress([]byte{3}): &ripemd160hash{},
+	common.BytesToAddress([]byte{4}): &dataCopy{},
+	common.BytesToAddress([]byte{5}): &bigModExp{},
+	common.BytesToAddress([]byte{6}): &bn256AddIstanbul{},
+	common.BytesToAddress([]byte{7}): &bn256ScalarMulIstanbul{},
+	common.BytesToAddress([]byte{8}): &bn256PairingIstanbul{},
+	common.BytesToAddress([]byte{9}): &blake2F{},
+
+	common.BytesToAddress([]byte{250}): &epoch{},
+	common.BytesToAddress([]byte{251}): nil,
```

Comment:
@MaxMustermann2 should this still be nil? Maybe we missed fixing this?

Reply:
Appears to be a UI error in the PR on GitHub as the file below looks good to me.

https://github.com/harmony-one/harmony/blob/181d90212cd27c0e7845d4a54f3d48b84e86f890/core/vm/contracts.go#L129-L135